src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -85,6 +85,35 @@ describe('store/modules/drafts', () => {
     });
   });
 
+  describe('#getAnnotationState', () => {
+    [
+      fixtures.draftWithTags,
+      fixtures.draftWithText,
+      fixtures.emptyDraft,
+    ].forEach(draftFixture => {
+      it('returns the draft for the annotation if one exists', () => {
+        store.createDraft(fixtures.annotation, draftFixture);
+        assert.deepEqual(
+          store.getAnnotationState(fixtures.annotation).annotation,
+          fixtures.annotation
+        );
+      });
+    });
+
+    it('returns object state based on annotation if no draft exists', () => {
+      assert.include(store.getAnnotationState(fixtures.annotation), {
+        text: fixtures.annotation.text,
+        tags: fixtures.annotation.tags,
+      });
+    });
+
+    it('sets `isPrivate` to false if no permissions are present', () => {
```

Comment:
Do you think we need a second `isPrivate ` test that also accounts for the isPrivate() imported method's truthy result?

`!!annotation.permissions && isPrivate(annotation.permissions)`